code/game/objects/items/plushes.dm

```diff
@@ -9,11 +9,52 @@
 	var/list/squeak_override //Weighted list; If you want your plush to have different squeak sounds use this
 	var/stuffed = TRUE //If the plushie has stuffing in it
 	var/obj/item/grenade/grenade //You can remove the stuffing from a plushie and add a grenade to it for *nefarious uses*
+	//--love ~<3--
+	var/male_chance = 1
+	var/female_chance = 1
+	var/obj/item/toy/plush/lover
+	var/obj/item/toy/plush/partner
+	var/obj/item/toy/plush/plush_child
+	var/obj/item/toy/plush/paternal_parent	//who initiated creation
+	var/obj/item/toy/plush/maternal_parent	//who owns, see love()
+	var/list/scorned	//who the plush hates
+	var/heartbroken = FALSE
+	var/vowbroken = FALSE
+	var/young = FALSE
+	var/mood_message = null
+	var/list/love_message
+	var/list/heartbroken_message
+	var/list/vowbroken_message
+	var/list/child_message
+	var/normal_desc
+	//--end of love :'(--
 
 /obj/item/toy/plush/Initialize()
 	. = ..()
 	AddComponent(/datum/component/squeak, squeak_override)
 
+/obj/item/toy/plush/New()
```

Comment:
put this in intialise, not new

Reply:
Okay will do. Can I ask why so I will hopefully remember in future?

Reply:
We have better control over when the Initialize proc runs, so we can better smooth out the creation of objects and manage it with our ticker subsystem, as opposed to new, which just runs straight away.